schemas/vertices/features.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "A component of a DNA sequence, such as a CDS, mRNA, etc.",
+  "required": [
+    "_key",
+    "type",
+    "location"
+  ],
+  "properties": {
+    "_key": {
+      "type": "string",
+      "title": "Content hash",
+      "description": "Hash of the DNA sequence for this feature."
+    },
+    "protein_translation": {
+      "type": "string",
+      "title": "Protein translation",
+      "description": "Longest coded protein (representative protein for splice variants)"
+    },
+    "protein_translation_length": {
+      "type": "integer",
+      "description": "Length of protein_translation"
+    },
+    "protein_hash": {
+      "type": "string",
+      "title": "Protein content hash",
+      "description": "Hash of the protein sequence that this feature encodes."
+    },
+    "note": {
+      "type": "string",
+      "description": "Free-text description of this feature"
+    },
+    "functions": {
+      "type": "array",
+      "title": "Gene functions",
+      "items": {"type": "string"}
+    },
+    "functional_descriptions": {
+      "type": "array",
+      "title": "Gene function descriptions",
+      "items": {"type": "string"}
+    },
+    "type": {
+      "type": "string",
+      "examples": ["Gene", "ncRNA", "repeat", "CDS", "mRNA"]
+    },
+    "location": {
+      "type": "array",
+      "description": "A list of segments of sequence that comprise this feature",
+      "items": {
+        "contig": {
+          "type": "string",
+          "description": "Contig ID where this segment occurs"
+        },
+        "strand": {
+          "type": "string",
+          "description": "Strand where this segment occurs",
+          "enum": ["+", "-", "?"]
+        },
+        "start": {
+          "type": "integer",
+          "description": "Index in the genome sequence where this segment of the feature starts"
+        },
+        "length": {
+          "type": "integer",
+          "description": "Length of this segment of the feature"
+        }
+      }
+    },
+    "flags": {
+      "type": "array",
+      "description": "Additional flags about the feature such trans_splicing",
+      "items": {"type": "string"}
+    },
+    "warnings": {
+      "type": "array",
+      "description": "Warnings generated by the uploader about this feature",
+      "items": {"type": "string"}
+    },
+    "dna_sequence": {
+      "type": "string",
+      "description": "Nucleotide sequence for this feature."
```

Comment:
This can blow up the document size. Leaving it in for now but we should consider dropping it.

Reply:
Maybe we can save an ID that points to a file store of a fasta?

Reply:
We can leave it optional and not import any of it for now